dashboard/test/models/ability_test.rb

```diff
@@ -83,6 +87,8 @@ class AbilityTest < ActiveSupport::TestCase
     assert !ability.can?(:destroy, Level)
     assert !ability.can?(:destroy, Activity)
 
+    assert !ability.can?(:read, Section)
```

Comment:
nit: any idea why some of these are `assert !ability.can?` and others are `assert ability.cannot?` ?

I see that you just followed the existing patterns. It would be nice to have consistency here. I'd prefer `cannot?` as in the CanCan canonical [example](https://github.com/ryanb/cancan/wiki/Testing-Abilities).